nevergrad/functions/control/core.py

```diff
@@ -0,0 +1,163 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+import numpy as np
+import typing as tp
+from nevergrad.parametrization import parameter as p
+from ..base import ExperimentFunction
+from .mujoco import GenericMujocoEnv
+
+
+class Ant(ExperimentFunction):
+    # Values from https://github.com/modestyachts/ARS/tree/master/trained_policies/Ant-v1
+    state_mean = [0.556454034343523, 0.9186531687987207, -0.003597273626800536, -0.062027209820968335, -0.22668151226915723, -0.16602131317263671, 0.7987064451368338, -0.08837934771162403, -0.5953573569489272, 0.31458301497655355, -0.5859666391196517, 0.06840799715850679, 0.6996188089127978, 2.957808943475944, 0.13145167715976466, 0.000408239775951425, 0.0006251405790721873, -0.01324649360933024, -0.010626814219628156, -0.0009340884177352034, 0.011235220933040631, -0.0037160116570636036, -0.018631124226896525, -0.00019431878659138357, -0.023557209761883236, 0.013354894059913155, 0.012594611303292104, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, -0.0002492521170156384, 5.2942688175970466e-06, 4.758743764016147e-06, 0.00023284707214249071, 0.000575762572920161, 0.008342223628535816, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, -3.805817666962799e-05, 2.944689402684725e-05, 4.622284386412991e-05, 2.0463934941826443e-05, -2.982578719158166e-07, 0.0003232789482430665, 0.01190863730406915, -0.028847450862012118, -0.0016049128587483268, -0.011546822405861629, -0.005185736733314698, 0.04395870932465012, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.00010306612506050966, 4.182155321673672e-05, -3.6599965153626765e-05, 3.7417588383893424e-05, -1.9852387226708935e-05, 0.0004990991660521935, 0.11191092578395258, 0.007451270486499179, -0.043879628923914234, 0.029746731314372638, 0.004893585846331603, 0.1289749574211565, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.00011265043476062721, -3.2651266270538316e-05, 4.192724068962384e-05, -3.599135906255506e-05, -5.0593649720061946e-05, 0.0005518224383528348, -0.06107825100454972, 0.06997376760122764, -0.0012712356710709012, -0.04546462254285545, -0.01656209815809629, 0.08931254442546284, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, -5.558307634083136e-05, -6.29738077240061e-05, -6.133269876741076e-05, -4.7529805786119285e-05, 2.6168802203125756e-05, 0.0003865204399435512, -0.08619662966579106, -0.07392835616333841, 0.06349519253475051, 0.04758718422237922, 0.014684068888490893, 0.10487554379090397]
```

Comment:
These numbers are used to normalize state values. I suspect it is done for stability and efficiency of the search. Would it work if we let them as optional parameters ?
To reproduce ARS and LaMCTS benchmarks in ``nevergrad.benchmark.experiments.control_problem``, one can fetch these values from corresponding link via http.What do you think ?

Reply:
Optional would not change anything if we keep those numbers in the code, http request seems a bit too complex.
From reading the License, it should be enough to add the corresponding License part https://github.com/modestyachts/ARS/blob/4c8e24e0a99cf811030e90680fc29eb94fae8cdd/LICENSE#L1-L23
to a ARS_LICENSE file in this folder, and mention this in `BaseFunction` docstring (that needs to be written, btw :p)
Optionally it may require stating in the README, but not sure that is necessary.
Also optionally, it may be a good practice to ask as an issue on the ARS code if that's fine with them, would you do it? Or I can do it as well.

Reply:
Great idea, let me do it :)